test/ex_unit_properties_test.exs

```diff
@@ -197,7 +197,7 @@ defmodule ExUnitPropertiesTest do
 
       check all int <- integer(),
                 max_runs: 25,
-                do: Agent.update(counter, &(&1 + 1)) && assert is_integer(int)
```

Comment:
Ehhh, wasn't assert in the default .formatter.exs.

Reply:
It is but we always parenthesize in some cases (I think this case is because the `assert` call is part of the `&&` call.